### PR TITLE
updated gemfile with gem versions

### LINKED ISSUE #46: Gems in gemfile aren't locked to any version in particular
For example:
```
gem 'devise'
```
should be
```
gem 'devise', '~> 4.2.0'
```

### PR DESCRIPTION
**Pull request by leoncalermo**
## Summary

- Updated the `Gemfile` with the gem versions
- Some gems was already with a version so i didn't change them, we can discuss each case.
- Fixes #46

### CODE REVIEW

**Comment by greysteil** [comment]
If you fancy keeping these versions up-to-date automatically (well, you get a PR each time there's a new version) you could use [Dependabot](https://dependabot.com). It's free for open-source.

File: Gemfile [hunk 1]
```diff
@@ -17,129 +17,129 @@ gem 'uglifier', '>= 1.3.0'
...
   # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
-  gem 'web-console'
+  gem 'web-console', '~> 3.5.1'
   # Spring speeds up development by keeping your application running in the background.
   # Read more: https://github.com/rails/spring
-  gem 'spring'
+  gem 'spring', '~> 2.0.2'
   gem 'spring-watcher-listen', '~> 2.0.0'
-  gem 'meta_request'
+  gem 'meta_request', '~> 0.4.3'
```
Comment (Dinuuu) [inline Gemfile]: Gem `meta_request` should appear before `spring-watcher-listen` in their gem group.

File: Gemfile [hunk 2]
```diff
@@ -17,129 +17,129 @@ gem 'uglifier', '>= 1.3.0'
...
+  gem 'bullet', '~> 5.6.1'
   gem 'listen', '~> 3.0.5'
   # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
-  gem 'web-console'
+  gem 'web-console', '~> 3.5.1'
   # Spring speeds up development by keeping your application running in the background.
   # Read more: https://github.com/rails/spring
-  gem 'spring'
+  gem 'spring', '~> 2.0.2'
+  gem 'meta_request', '~> 0.4.3'
```
Comment (Dinuuu) [inline Gemfile]: Gem `meta_request` should appear before `spring` in their gem group.

Check that gems should be displayed alphabetically within its group

**Comment by alebian** [comment]
Perhaps it would be a good time to add [my Rubocop rule](https://gist.github.com/alebian/9a980f514001ae62b46d5469e5bb92c8) to avoid gems without versions. What do you think? @Dinuuu @hdf1986 @mishuagopian @icoluccio @francoprud

**Comment by hdf1986** [comment]
@alebian agree, do you remember where's the snippet?

**Comment by mishuagopian** [comment]
Are we updating this? @Dinuuu @alebian @hdf1986 @leoncalermo

**Pull request closed without merging**